frontend/src/modules/EditZing/Components/NotesModal.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import {
+  Modal,
+  Box,
+  Button,
+  Typography,
+  OutlinedInput,
+  IconButton,
+} from '@mui/material'
+import LoadingButton from '@mui/lab/LoadingButton'
+import SaveIcon from '@mui/icons-material/Save'
+import CloseIcon from '@mui/icons-material/Close'
+
+import { API_ROOT } from '@core/Constants'
+import axios from 'axios'
+
+const NotesModal = (props: any) => {
+  const { courseId } = useParams<{ courseId: string }>()
+  const { open, student, handleClose, setSaved, setNotSaved } = props
+  const [note, setNote] = useState(student.notes)
+  const [saving, setSaving] = useState(false)
+  const email = student.email
+
+  // getting notes for student
+  useEffect(() => {
+    const savedNote = student.groups.find((g: any) => g.courseId === courseId)
+      .notes
+    setNote(savedNote)
+  }, [courseId, student])
+
+  // saving notes to db
+  const handleSave = async () => {
+    setSaving(true)
+    await axios
+      .post(`${API_ROOT}/student/notes`, {
+        email: email,
+        courseId: courseId,
+        notes: note,
+      })
+      .then(() => setSaved(true))
+      .catch(() => setNotSaved(true))
+    setSaving(false)
+    handleClose()
+  }
+
+  return (
+    <Modal
+      open={open}
+      onClose={handleClose}
+      sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}
+    >
```

Comment:
@mluo24 made a ZingModal component that's in use in other parts of the app like the email modal. How does it compare to using a plain MUI Modal here?

Reply:
Yeah, I mean I think a regular modal is just simpler to use.

Reply:
how, you're writing more lines of code ;-;